src/wallet/wallet.cpp

```diff
@@ -3298,7 +3298,7 @@ bool CWallet::InitLoadWallet()
             CKey key;
             key.MakeNewKey(true);
             if (!walletInstance->SetHDMasterKey(key))
-                throw std::runtime_error("CWallet::GenerateNewKey(): Storing master key failed");
+                throw std::runtime_error("CWallet::InitLoadWallet(): Storing master key failed");
```

Comment:
Fine with me. I think it would help if someone added a short sentence to https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md about returning a bool vs throwing an exception.